dpgen/generator/run.py

```diff
@@ -2503,20 +2503,29 @@ def post_fp_cp2k (iter_index,
     system_index.sort()
 
     cwd = os.getcwd()
+    tcount = 0
+    icount = 0
     for ss in system_index :
         sys_output = glob.glob(os.path.join(work_path, "task.%s.*/output"%ss))
         sys_output.sort()
-        for idx,oo in enumerate(sys_output) :
-            sys = dpdata.LabeledSystem(oo, fmt = 'cp2k/output')
-            if len(sys) > 0:
-                sys.check_type_map(type_map = jdata['type_map'])
-            if idx == 0:
-                all_sys = sys
+        tcount += len(sys_output)
+        all_sys = None
+        for oo in sys_output :
+            _sys = dpdata.LabeledSystem(oo, fmt = 'cp2k/output')
+            if len(_sys) == 1:
```

Comment:
dpdata should support appending empty system, so you don't need to make a if-statement...

Reply:
I think this is used for counting failed ratio?

Reply:
I think it's more clear to use `len(sys_output)-len(all_sys)` instead of the current way.

Reply:
> I think it's more clear to use `len(sys_output)-len(all_sys)` instead of the current way.

I adapt this way